src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -231,6 +231,13 @@ const SettingsModalStackNavigator = createModalStackNavigator([
         },
         name: 'Settings_Password',
     },
+    {
+        getComponent: () => {
+            const SettingsPasswordPage = require('../../../pages/settings/ResendValidationLinkScreen').default;
```

Comment:
This component is in `pages/settings` so let's change the name to Page instead of Screen
```suggestion
            const SettingsPasswordPage = require('../../../pages/settings/ResendValidationLinkPage').default;
```